### DISCUSSION TITLE
Plans to add diarization?

### DISCUSSION THREAD

**Discussion by JusticeGH** [Purfview/whisper-standalone-win#234] [Q&A]
Curious if this is something that's either a hidden setting or planned to be added, if at all?

**Comment by Purfview** [comment]
> Curious if this is something that's either a hidden setting or planned to be added, if at all?

Currently there is no diarization implemented. Could be added to Standalone Faster-Whisper-XXL.
Is there diarization implementation using onnx models?

**Reply by ghost** [reply]
IIRC, there were some forks for pyannote onnx version.

**Reply by Purfview** [reply]
>IIRC, there were some forks for pyannote onnx version.

But there I can see only VAD.

**Reply by JusticeGH** [reply]
The only thing I'm seeing rn is: https://github.com/pengzhendong/pyannote-onnx

**Reply by Purfview** [reply]
I don't see any diarization in the code there

**Reply by tristan-mcinnis** [reply]
First, thanks for putting this together - it works like a charm. Think if there was diarization, then it would be perfect. 
I guess for, ill try doing the diarization separately and then add it back into the transcript?
